core-macros/src/main/scala/sbt/internal/util/appmacro/ContextUtil.scala

```diff
@@ -1,15 +1,14 @@
 package sbt.internal.util.appmacro
 
-import sbt.internal.util.Types.Id
 import scala.compiletime.summonInline
 import scala.quoted.*
-import scala.reflect.TypeTest
 import scala.collection.mutable
 import sbt.util.cacheLevel
 import sbt.util.CacheLevelTag
 
-trait ContextUtil[C <: Quotes & scala.Singleton](val qctx: C, val valStart: Int):
+trait ContextUtil[C <: Quotes & scala.Singleton](val valStart: Int):
   import qctx.reflect.*
+  val qctx: C
```

Comment:
I am guessing that the `val` should come before the `import` line above it.

Reply:
Probably the compiler can understand his because the val is abstract. But yes it would be clearer to move the `import` after the `val`.